csm/core/agent/csm_agent.py

```diff
@@ -135,6 +136,18 @@ def _configure_s3_services():
         CsmRestApi._app[const.S3_BUCKET_SERVICE] = BucketService(s3_plugin_obj)
         CsmRestApi._app[const.S3_CAPACITY_SERVICE] = S3CapacityService(s3_plugin_obj)
 
+    @staticmethod
+    def get_topology_config():
+        host = Conf.get(const.DATABASE_INDEX, const.CONSUL_CONFIG_HOST)
```

Comment:
This seems repetitive

Reply:
Can we skip get call and use a variable already loaded?

Reply:
removed get call